core/arch/aarchxx/emit_utils.c

```diff
@@ -50,7 +50,7 @@ insert_load_dr_tls_base(dcontext_t *dcontext, instrlist_t *ilist, instr_t *where
     /* Load TLS base from user-mode thread pointer/ID register:
      * mrs reg_base, tpidr_el0
      */
-    PRE(ilist, where,
+    PRE(ilist, whre,
```

Comment:
This and below are supposed to be temporary do-not-check-in I assume -- I would suggest always having a comment saying that as a safe practice.

Reply:
Will do in future. Thanks.

Reply:
Reverted these temporary changes.